docs2017/src/api/attachments.md

```diff
@@ -64,7 +64,7 @@ Content-Type: $$ATTACHMENT_MIME_TYPE
 _Example instruction for creating or updating an attachment, using the command line:_
 
 ```sh
-curl https://$USERNAME:$PASSWORD@$USERNAME.cloudant.com/$DATABASE/$DOCUMENT_ID/$ATTACHMENT?rev=$REV \
+curl https://$ACCOUNT:$PASSWORD@$ACCOUNT.cloudant.com/$DATABASE/$DOCUMENT_ID/$ATTACHMENT?rev=$REV \
```

Comment:
remove `$ACCOUNT:$PASSWORD@` from this (`curl` examples should exclude authentication except when explicitly demonstrating how to authenticate)